### PR TITLE
Update Infra Admin Bot documentation to reflect changes in IRC configuration

### PR DESCRIPTION
**Pull request by oleg-nenashev**
The bot configuration change was requested in the chat by @timja and @olblak , because the password authentication has expired and impacted the IRC integration.

CC @timja @olblak

### CODE REVIEW

File: bots.adoc [hunk 1]
```diff
@@ -37,10 +37,13 @@ Ex:
...
 
 
-This bot is used to for various infrastructure task that required Github integration
+This bot is used to for various infrastructure task that required Github integration.
+It is also used for integration GitHub org management.
+
 ex:
 
 * pluginsite configuration
+* jenkinsadmin [IRC bot](https://www.jenkins.io/projects/infrastructure/ircbot/)
```
Comment (timja) [inline bots.adoc:46]: doesn't seem to be rendering properly, i think this is the syntax

```suggestion
* jenkinsadmin link:https://www.jenkins.io/projects/infrastructure/ircbot/[IRC bot]
```

Comment (timja) [reply bots.adoc:46]: or
```suggestion
* jenkinsadmin https://www.jenkins.io/projects/infrastructure/ircbot/[IRC bot]
```

**Review by timja — APPROVED** [review]

**Comment by timja** [comment]
can the 'core' team be added with write access here please @olblak

**Comment by olblak** [comment]
> can the 'core' team be added with write access here please @olblak

Done

File: bots.adoc [hunk 2]
```diff
@@ -59,7 +62,6 @@ This bot is the first GitHub bot used in the Jenkins Infrastructure.
 ex:
 
 * Automatically open pull request
-* jenkinsadmin irc bot
```
Comment (olblak) [inline bots.adoc:62]: We replaced the gihtub account [jenkinsadmin](https://github.com/jenkinsadmin) by [jenkins-infra-bot](https://github.com/jenkins-infra-bot)

Comment (olblak) [reply bots.adoc:62]: I am not sure that jenkinsadmin irc bot is refering to, the github account? the irc account ? jenkins-admin

Comment (timja) [reply bots.adoc:62]: the irc account is what I assumed it meant

Comment (oleg-nenashev) [reply bots.adoc:62]: IIRC it was a historical doc I just moved a while ago

**Review by markyjackson-taulia — APPROVED** [review]

**Pull request merged**